django_airavata/apps/admin/static/django_airavata_admin/src/components/statistics/ExperimentStatisticsContainer.vue

```diff
@@ -41,6 +41,7 @@
                     ></i>
                   </b-input-group-prepend>
                   <flat-pickr
+                    :maxDate="tomorrow"
```

Comment:
I don't see `maxDate` as an option https://github.com/ankurk91/vue-flatpickr-component, this has to be passed in via the config, which the code already does.

Reply:
@machristie Yes, you are right. Thank you for catching this error. I have removed the extra code related to the maxDate binding.